src/compositor/hwcimage.cpp

```diff
@@ -66,7 +66,7 @@ class HwcImageLoadRequest : public QRunnable, public QEvent
             // Apply glass..
             if (effect.contains(QStringLiteral("glass"))) {
                 QImage glass("//usr/share/themes/jolla-ambient/meegotouch/icons/graphic-shader-texture.png");
-                glass = glass.scaled(glass.width() * pixelRatio, glass.height() * pixelRatio,
```

Comment:
We'll make sure the bitmap is scaled properly so that we don't need to do upscaling here.